ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java

```diff
@@ -367,9 +367,8 @@ public static void setRole(
     } else if (role.equals(AccessibilityRole.SUMMARY)) {
       nodeInfo.setRoleDescription(context.getString(R.string.summary_description));
     } else if (role.equals(AccessibilityRole.HEADER)) {
-      nodeInfo.setRoleDescription(context.getString(R.string.header_description));
```

Comment:
This change breaks accessibilityRole='header' on Android for RN 63.x. It was working fine on 62.2

Reply:
Yep! That's right! I'm having this issue on my RN app.